src/Schema/AbstractSchemaManager.php

```diff
@@ -211,6 +211,29 @@ public function listTableColumns($table, $database = null)
         return $this->_getPortableTableColumnList($table, $database, $tableColumns);
     }
 
+    /**
+     * @param string      $table
+     * @param string|null $database
+     *
+     * @return Column[]
```

Comment:
Why not use native type declarations for this?

Reply:
For return types, yes, makes sense.